### ISSUE TITLE
A working system based on the latest release 5.10.2 ?

### ISSUE THREAD

**Issue by Phaengris** [jekhor/yogabook-linux#20]
The device is YB1-X91F.

I followed this instruction https://github.com/jekhor/yogabook-linux/wiki
so I got the working system, but without the features of the last release.

Then I tried this:
- install Debian Sid (as I see the latest livecd is based on it)
- download and install the last release assets https://github.com/jekhor/yogabook-linux/releases/tag/livecd-5.10.2

Still no luck:
- autorotation isn't applied when the system starts (but it starts to work if I lock screen, then unlock it)
- sleep/resume doesn't work properly (after resume the tablet screen is black; though kbd/wacom works and an external monitor works too)
- mouse button pads doesn't work on the halo kbd (mouse pad itself responds and moves the cursor, but the side pads don't act as mouse buttons)
- wifi doesn't work (might be a Debian firmware issue though)

I confirm when I boot the livecd, there are no issues mentioned above.
- autorotation works and is applied right on the login screen
- sleep/resume works properly
- wifi works
(not sure about the mouse button pads, didn't test it with livecd)

So is there any way to build an working system with the last release features? Make them work right as they do with livecd?

Thanks.

**Comment by jekhor** [comment]
Hmmm...
I have Debian sid installed on my Yoga Book with released debian packages and all things work...

What desktop environment do you use? I got autorotation working with Gnome 3 only. Run the monitor-sensor command and check that position is reported properly when device is rotated. If no, write here and attach the 'journalctl -b' and 'dmesg' commands output, if possible.

Mouse keys at sides of touchpad were not configured, I use two/three fingers' tap. But it should be enough to add them to the layout.csv file as keys (touchpad size may need correction also).

For wi-fi check the dmesg and try to find needed firmware package (firmware-brcm80211, as I remember).

No ideas about sleep/resume, try to check dmesg after resume...

**Comment by jules6969** [comment]
So I've got similar problems.
Running Ubuntu 20.10 on YB1-X91F and also followed the guide. After resume the holo keyboard works but doesn't light up, the screen works fine. I had to install the right version of android-libbase because apt get installs version 1.8.. instead of 1.10.. (But I think everyone could figure that out by themselves). 
My Screen rotation also just works on the login menu not in Gnome3 Desktop.
I had no Problems with WiFi connection and neither Bluetooth.

But I appreciate your work and wanted to thank you a lot!

**Comment by mathewjpotter** [comment]
I agree with Jules, I did follow the instructions, default install of ubuntu 20.10. The keyboard doesn't light up after sleep, seems a bit slower than it did no on the usb. screen rotation doesn't work for me automatically, had to set it.

that being said, I can't tell you how much I appreciate your efforts, here is my output 
[journal.txt](https://github.com/jekhor/yogabook-linux/files/6279669/journal.txt)
[dmesg (1).txt](https://github.com/jekhor/yogabook-linux/files/6279670/dmesg.1.txt)
